setup.py

```diff
@@ -34,6 +34,9 @@
     license="Apache 2.0",
     python_requires='>=3.8',
     install_requires=install_requires,
+    extras_require={
+        'data': ["pycocotools=2.0.6"],
```

Comment:
can we have this as `pip install "clarifai[all]"` or  this is fine `pip install "clarifai[data]"` ?

Reply:
Both seems fine, since data-utilities are planned next, I thought of having `data`

Reply:
@sriram-clarifai your thoughts please